scrapers_next/ar/people.py

```diff
@@ -131,3 +131,235 @@ def process_item(self, item):
         p = PartialMember(name=name, chamber=chamber, url=self.source.url)
 
         return LegDetail(p, source=source)
+
+
+class SenDetail(HtmlPage):
+    example_source = "https://senate.arkansas.gov/senators/missy-irvin/"
+
+    def process_page(self):
+        rows = XPath("//div[contains(@class,col-md-8)]/ul/li").match(self.root[0])
+
+        image_container = CSS(".col-md-4").match(self.root)[0]
+        image = (
+            XPath("//img[contains(@alt, '" + self.input.name + "')]")
+            .match(image_container)[0]
+            .get("src")
+        )
+        image = image.split("?")[0]
+
+        biocontainer = CSS(".panel-body").match(self.root)[0]
+        bio = XPath("//a/@href").match(biocontainer)[0]
+        committees = XPath("//div[contains(@class,base-text)]/ul/li").match(self.root)
+
+        table = {
+            "Party": "",
+            "Phone": "",
+            "Email": "",
+            "District": "",
+            "Seniority": "",
+            "Occupation": "",
+            "District Address": "",
+            "Church Affiliation:": "",
+            "Veteran": "",
+            "Public Service": "",
+            "Legislative Service": "",
+            "Biography": "",
+        }
+
+        for row in rows:
+            try:
+                text = row.text_content()
+                text = text.split(": ")
+                if len(text) < 2:
+                    continue
+                item_name = text[0]
+                value = text[1]
+                table[item_name] = value
+                if item_name == "Legislative Service":
+                    if value[0] != "H":
+                        value = value.replace("House", "; House")
+                    table[item_name] = value
+                    break
+
+            except SelectorError:
+                pass
+
+        table["Biography"] = bio
+
+        p = ScrapePerson(
+            name=self.input.name,
+            state="ar",
+            chamber=self.input.chamber,
+            party=table["Party"],
+            image=image,
+            district=table["District"],
+            email=table["Email"],
+        )
+
+        committees_string = ""
+        first_committee = True
+        for c in committees:
+            found = False
+            for k in table.keys():
+                if c.text_content().find(k) >= 0:
+                    found = True
+            if not found:
+                if not first_committee:
+                    committees_string += ", "
+                committees_string += c.text_content().lower()
+                first_committee = False
+        p.extras["Committees"] = committees_string
+
+        if table["District Address"] != "":
```

Comment:
`if not table["District Address"]:` catches the more general "empty" case. Should we use that?